public/video-ui/src/components/Workflow/Workflow.js

```diff
@@ -48,6 +48,11 @@ class Workflow extends React.Component {
         workflowStatuses={this.props.workflow.statuses || []}
         workflowPriorities={this.props.workflow.priorities}
         workflowStatus={this.props.workflow.status}
+        workflowProductionOffices={[
+          { id: 'UK', title: 'UK' },
+          { id: 'US', title: 'US' },
+          { id: 'AU', title: 'AU' }
+        ]}
```

Comment:
Setting this in here as it's where the other properties get set. Hardcoding because that's what happens in Composer and workflow itself